api/v1beta1/hyperconverged_types.go

```diff
@@ -356,6 +356,15 @@ type HyperConvergedFeatureGates struct {
 	// +kubebuilder:default=false
 	// +default=false
 	DisableMDevConfiguration *bool `json:"disableMDevConfiguration,omitempty"`
+
+	// Enable persistent reservation of a LUN through the SCSI Persistent Reserve commands on Kubevirt.
+	// In order to issue privileged SCSI ioctls, the VM requires activation of the persistent reservation flag.
+	// Once this feature gate is enabled, then the additional container with the qemu-pr-helper is deployed inside the virt-handler pod.
+	// Enabling (or removing) the feature gate causes the redeployment of the virt-handler pod.
+	// +optional
+	// +kubebuilder:default=false
+	// +default=false
+	PersistentReservation *bool `json:"persistentReservation,omitempty"`
```

Comment:
@alicefr What are the risks for having this always enabled?  I'm always in favor of fewer knobs.  Is there a good reason that some people would like to have this off?  If so, should the default be on or off?

Reply:
If we don't see real reasons why users should avoid it, I'd also prefer to have it as an opinionated value.
I started filing https://github.com/kubevirt/hyperconverged-cluster-operator/pull/2374 as an alternative to have this as an opinionated value.